src/EnergyPlus/VentilatedSlab.cc

```diff
@@ -337,26 +322,27 @@ namespace VentilatedSlab {
             state.dataVentilatedSlab->VentSlabNumericFields(Item).FieldNames.allocate(NumNumbers);
             state.dataVentilatedSlab->VentSlabNumericFields(Item).FieldNames = cNumericFields;
             UtilityRoutines::IsNameEmpty(state, state.dataIPShortCut->cAlphaArgs(1), CurrentModuleObject, ErrorsFound);
+            auto & ventSlab = state.dataVentilatedSlab->VentSlab(Item);
 
-            state.dataVentilatedSlab->VentSlab(Item).Name = state.dataIPShortCut->cAlphaArgs(1);
-            state.dataVentilatedSlab->VentSlab(Item).SchedName = state.dataIPShortCut->cAlphaArgs(2);
+            ventSlab.Name = state.dataIPShortCut->cAlphaArgs(1);
+            ventSlab.SchedName = state.dataIPShortCut->cAlphaArgs(2);
             if (lAlphaBlanks(2)) {
-                state.dataVentilatedSlab->VentSlab(Item).SchedPtr = DataGlobalConstants::ScheduleAlwaysOn;
+                ventSlab.SchedPtr = DataGlobalConstants::ScheduleAlwaysOn;
             } else {
```

Comment:
Thing 2. Here is a shorthand:

```cpp
else if ((ventSlab.SchedPtr = GetScheduleIndex(state, state.dataIPShortCut->cAlphaArgs(2)) == 0) {
      ShowSevereError ...
```

Basically, you can combine the assignment and the check of that assignment into one line.